spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkWriteOptions.java

```diff
@@ -68,4 +68,10 @@ private SparkWriteOptions() {
   // Controls whether to take into account the table distribution and sort order during a write operation
   public static final String USE_TABLE_DISTRIBUTION_AND_ORDERING = "use-table-distribution-and-ordering";
   public static final boolean USE_TABLE_DISTRIBUTION_AND_ORDERING_DEFAULT = true;
+
+  // Identifies snapshot from which to start validating conflicting changes
+  public static final String VALIDATE_FROM_SNAPSHOT_ID = "validate-from-snapshot";
+
+  public static final String DYNAMIC_OVERWRITE_ISOLATION_LEVEL = "write.dynamic.overwrite.isolation-level";
```

Comment:
We try to use short names for read and write options without dots to indicate namespace as opposed to table props. This option follows the table property convention.

What about making this property generic? We could call it `isolation-level` and then respect in DELETE, UPDATE, MERGE operations too (in a separate PR).

Reply:
Good point, updated